### PR TITLE
Disable Popover GPU acceleration on Windows.

### PR DESCRIPTION
**Pull request by RobertBolender**
See also:

1. Earlier PR #28, 
2. Associated Bootstrap Bug twbs/bootstrap#22610.

### CODE REVIEW

File: components/popover/popover-base.jsx [hunk 1]
```diff
@@ -143,10 +144,22 @@ export class PopoverBase extends Component {
 		} = this.props;
 		const { showPopper } = this.state;
 
+		const popperModifiers = {
+			...modifiers,
+			computeStyle: {
+				...modifiers.computeStyle,
+				gpuAcceleration: !(
+					typeof window !== `undefined` &&
```
Comment (bryanrsmith) [inline components/popover/popover-base.jsx]: Please move move `window` access to a lifecycle method.

Comment (RobertBolender) [reply components/popover/popover-base.jsx]: Is that still necessary, even though I'm checking whether the window is undefined?

File: components/popover/popover-base.jsx [hunk 2]
```diff
@@ -74,11 +74,13 @@ export class PopoverBase extends Component {
 		theme: {
 			backgroundColor: colors.white,
 		},
+		modifiers: {},
 		styleOverrides: {},
 	};
 
 	state = {
 		showPopper: this.props.isOpen || false,
+		popperModifiers: this.props.modifiers,
```
Comment (RobertBolender) [inline components/popover/popover-base.jsx]: I realize that copying props into state is almost never a good idea.
I don't expect consumers to change this `modifiers` prop, are we comfortable with this tradeoff, and does it need to be explicitly called out in a comment or something?

Comment (smaxa20) [reply components/popover/popover-base.jsx]: Why is it not expected that consumers will change the `modifiers` prop? And if it's not expected to be used why is it included as a prop?

Comment (RobertBolender) [reply components/popover/popover-base.jsx]: I should clarify: I don't expect that a consumer would change the prop between renders, so we would be OK with ignoring updates to the prop and only use the initial value that is passed in.

Comment (smaxa20) [reply components/popover/popover-base.jsx]: Oh ok that makes more sense. Thank you.

Comment (bryanrsmith) [reply components/popover/popover-base.jsx]: We could avoid the problem by only putting a `shouldDisableGPUAcceleration` flag in state, and merging the necessary modifiers with the modifiers specified by the consumer at render time.

...Is it worth any of this complication? Should we disable it unconditionally?

Comment (RobertBolender) [reply components/popover/popover-base.jsx]: I'm not opposed to disabling GPU acceleration unconditionally.
Without objection, doing that.

**Pull request merged**